hack/copy-kube-artifacts.sh

```diff
@@ -16,49 +15,30 @@ fi
 
 # Copy special files.
 rsync -av \
-  --exclude='examples/blog-logging/diagrams/***' \
-  --exclude='pkg/ui/data/swagger/datafile.go' \
+  --exclude='BUILD' \
   --include-from=- \
   --include='*/' \
   --exclude='*' \
   --prune-empty-dirs \
   $KUBE_ROOT/ $KUBE_GODEP_ROOT <<EOF
 /api/swagger-spec/*.json
-/cmd/integration/***
```

Comment:
Do we need them? If so we should godep save them rather. Besides this script needs more love post-rebase, so that it updates `Godeps.json` when adding stuff from k8s, otherwise we're getting mess :/